db/seeds.rb

```diff
@@ -47,18 +47,37 @@
 
 Topic.destroy_all
 
-topic1 = Topic.new(
-    topic: "BLM"
-)
+blm = Topic.create(topic: "BLM")
+lgbtq = Topic.create(topic: "LGBTQ+")
+legal = Topic.create(topic: "Legal")
+physical_health = Topic.create(topic: "Physical Health")
+mental_wellbeing = Topic.create(topic: "Mental Wellbeing")
+immediate_danger = Topic.create(topic: "Immediate Danger")
+domestic_issues = Topic.create(topic: "Domestic Issues")
+general = Topic.create(topic: "General")
```

Comment:
This does feel a bit repetitive; so if you plan to create more topics in the future you might want to consider DRY-ing this up.